### PR TITLE
Added Equidistant Cylindrical Projection and test

### PR DESCRIPTION
**Pull request by jproust**

### CODE REVIEW

File: Source/udGeoZone.cpp [hunk 1]
```diff
@@ -1847,7 +1873,7 @@ static double udGeoZone_DelambreCoefficients(const double e)
 {
   // First Delambre coeficients Ao
   // http://www.mygeodesy.id.au/documents/Meridian%20Distance.pdf p.7
-  return 1.0 - 1.0 / 4.0 * udPow(e, 2) - 3.0 / 64.0 * udPow(e, 4) - 5.0 / 256.0 * udPow(e, 6) - 175.0 / 16384.0 * udPow(e, 8) - 441.0 / 65536.0 * udPow(e, 10);
+  return 1.0 - 1.0 / 4.0 * udPow(e, 2) - 3.0 / 64.0 * udPow(e, 4) - 5.0 / 256.0 * udPow(e, 6) - 175.0 / 16384.0 * udPow(e, 8) - 441.0 / 65536.0 * udPow(e, 10) - 4851.0 / 1048576.0 * udPow(e, 12) - 14157.0 / 4194304.0 * udPow(e, 14);
```
Comment (OsiJr) [inline Source/udGeoZone.cpp:1876]: The extra precision here didn't break anything?

File: udTest/src/udGeoZoneTests.cpp [hunk 2]
```diff
@@ -472,7 +472,7 @@ TEST(udGeoZone, HotineObliqueMercator)
   EXPECT_EQ(udRound(latLong.y * localPrecision), udRound(latLongRes.y * localPrecision));
 }
 
-TEST(udGeoZone, AlbersConicEqualArea)
+  TEST(udGeoZone, AlbersConicEqualArea)
```
Comment (OsiJr) [inline udTest/src/udGeoZoneTests.cpp:475]: Extra whitespace

File: udTest/src/udGeoZoneTests.cpp [hunk 3]
```diff
@@ -488,6 +488,21 @@ TEST(udGeoZone, AlbersConicEqualArea)
   udDouble3 pos = udGeoZone_LatLongToCartesian(geoZone, latLong);
   EXPECT_EQ(udRound(pos.x * localPrecision), udRound(1466493.492 * localPrecision));
   EXPECT_EQ(udRound(pos.y * localPrecision), udRound(702903.006 * localPrecision));
+}
+  TEST(udGeoZone, EquidistantCylindrical)
```
Comment (OsiJr) [inline udTest/src/udGeoZoneTests.cpp:492]: Whitespace and line break

File: Source/udGeoZone.cpp (hunk 1 above)
Comment (jproust) [reply Source/udGeoZone.cpp:1876]: no

Comment (jproust) [reply Source/udGeoZone.cpp:1876]: it just add precision

File: udTest/src/udGeoZoneTests.cpp (hunk 2 above)
Comment (jproust) [reply udTest/src/udGeoZoneTests.cpp]: Fixed

File: udTest/src/udGeoZoneTests.cpp (hunk 3 above)
Comment (jproust) [reply udTest/src/udGeoZoneTests.cpp]: Fixed

**Review by OsiJr — APPROVED** [review]

**Pull request merged**